src/components/SocialButton/index.tsx

```diff
@@ -39,7 +42,7 @@ export const SocialButton = ({
       overloadLoginCb = false,
     } = hookConfig;
 
-    const res = HOOKS_MAPPER[name](args);
+    const res = HOOKS_MAPPER[name]?.(args);
 
     if (overloadLoginCb) loginCb = res;
```

Comment:
If args is no more required, we must check also that res is not undefined in order to overload loginCb